### DISCUSSION TITLE
Group field : add option to set open by default

### DISCUSSION THREAD

**Discussion by neo7-studio-web** [directus/directus#6588] [Feature Requests]
Currently a group field is closed by default. Would it be possible to have an option to set it open by default ?

**Comment by benhaynes** [comment]
Absolutely! I think there should be Interface Options for:

* Allow Collapsing
* Initial State
    * Start Open
    * Start Collapsed
* Accordion — Close other groups when opening

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
